### ISSUE TITLE
Clear performance improvement opportunities

### ISSUE THREAD

**Issue by Eiim** [Paradox2102/roborecon-app#4]
I ran an audit in Chrome's DevTools on roborecon.net/group-matches because I felt that the website was rather slow. It scored a 0/100 in performance, so I thought I'd look a little closer. The two main issues it found were the Open Sans font and text compression. The text compression would have to be handled server-side, and I don't know your setup, but the font should be a much simpler fix: by using <link rel=preload> on the most important requests (JS, CSS, images), the rest of the webpage can load before the fonts, so that you aren't waiting for the font files to load. Chrome estimates that this would save as much as 17.25 seconds of page load. I'm... skeptical of that, but it should help some. I couldn't find anything mentioning the font in this repo though. Is there something I'm missing?

Also, gzip compression might be even better, but again, no idea how your backend is working.

**Comment by runninbear5** [comment]
We have actually updated our application completely with a new url and new functionality.  The url is https://roborecon.net/

We now give users the capability to create groups and scout specific events. It is also running online permanently, not requiring a team to host their own site.

Let me know if you have any questions about our new application or any suggestions.

**Comment by Eiim** [comment]
Yes, roborecon.net is the domain I was talking about. Apologies if this is the wrong repo.
